foundations/installations/setting_up_git.md

```diff
@@ -73,7 +73,15 @@ This will install the latest version of Git. Easy, right?
 git --version
 ~~~
 
-If the version number is less than 2.28, follow the instructions again. If you are encountering a `no formulae found in taps` error:
+If the version number is less than 2.28, 
+
+~~~bash
+brew upgrade git
+~~~
+
+If this does not work, follow instructions again.
+
+If you are encountering a `no formulae found in taps` error:
```

Comment:
Thanks for contributing! I spoke with the team after getting some clarification about the installation process, and we're of the opinion that if Git didn't update with `brew install git`, because by default it will attempt to install the latest version, we are assuming `brew update git` is not going to do anything and that we should probably want to go to `brew doctor`.  

With that in mind, we can probably update this to instead say "If you are encountering a  `no formulae found in taps` error, or your version is still less than 2.28:" so we can catch those cases.

Reply:
Are you still interested in making this update?

Reply:
I am still interested in making this update. 
I had this particular issue when I was unable to fetch an update for git with `brew install git`. I ran into errors, which was mitigated with `brew upgrade git`.
Please note `brew update` and `brew upgrade` are different functions, and are not compatible with each other in my experience.

I feel that the beginner git user might not understand a `no formulae found in taps` error and I suggest the command should make it easier.

Please revert back to discuss further changes.